examples/enable-operator-and-auto-instrumentation/enable-operator-and-auto-instrumentation-values.yaml

```diff
@@ -4,6 +4,8 @@ splunkObservability:
   logsEnabled: true
   accessToken: CHANGEME
   realm: us0
+  # Enables Always on Profiling
+  # profilingEnabled: true
```

Comment:
Why is this commented out? Shouldn't it be applied?

Reply:
From recent directions, profiling is not enabled by default when enabling auto-instrumentation. This is an opt-in after option, was trying to show this exist here with a comment. I'm trying to add "custom sub-case manifest renders" in the examples that will show off options like "profilingEnabled: true" well in a future PR. Let me know if you have input.

Reply:
Removed these lines since we document them [instrumentation-enable-profiling.yaml](https://github.com/signalfx/splunk-otel-collector-chart/pull/836/files#diff-d0ee0bbabe39234f7ecfcbca2879598f013e0f3209463169c18984618bf5ada4)